### PR TITLE
[CARBONDATA-1661] Fixed bug related to display of timestamp values in presto

### PR DESCRIPTION
**Pull request by geetikagupta16**

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/788/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1421/

File: integration/presto/src/main/java/org/apache/carbondata/presto/PrestoFilterUtil.java [hunk 1]
```diff
@@ -258,6 +258,8 @@ else if (type instanceof DecimalType) {
         return new BigDecimal(new BigInteger(String.valueOf(rawdata)),
             ((DecimalType) type).getScale());
       }
+    } else if (type.equals(TimestampType.TIMESTAMP)) {
```
Comment (jackylk) [inline integration/presto/src/main/java/org/apache/carbondata/presto/PrestoFilterUtil.java]: You can use `type == DataTypes.TIMESTAMP` instead

Comment (geetikagupta16) [reply integration/presto/src/main/java/org/apache/carbondata/presto/PrestoFilterUtil.java]: Here I am comparing the type with Presto's datatypes that's why I have used `TimestampType.TIMESTAMP`

**Comment by chenliang613** [comment]
verified. LGTM

**Pull request closed without merging**